internal/watcher/validator.go

```diff
@@ -143,16 +144,15 @@ func (u *Updatable) Update() error {
 		if err != nil {
 			return err
 		}
-		idkeydigest, err = hex.DecodeString(string(idkeydigestRaw))
-		if err != nil {
-			return fmt.Errorf("parsing hexstring: %s: %w", idkeydigestRaw, err)
+		if err = json.Unmarshal(idkeydigestRaw, &digest); err != nil {
+			return fmt.Errorf("unmarshaling content of IDKeyDigestFilename: %s: %w", idkeydigestRaw, err)
 		}
-		u.log.Debugf("New idkeydigest: %x", idkeydigest)
+		u.log.Debugf("New idkeydigest: %v", digest)
```

Comment:
Until v2.6.0 this function, in case the `idkeydigests` file is not found, should try to read from the older `idkeydigest` file, and use it to create the `idekeydigests` file in the new format.

Reply:
I don't think that we need to build a migration for this, as there is no upgrade path currently between minor versions in Constellation. Therefore, creating a new ConfigMap seems a bit much. Maybe it is enough for the joinservice to also consume the old file?

Reply:
Mostly I want this so we don't start neglecting upgrade compatibility just because "we are not quite there yet".
Thinking about this again, just being able to read the old file should be sufficient.
Maybe this won't even be required, since helm should recreate the relevant configmap during an upgrade, but I'm not 100% sure on this.

Reply:
I was thinking only about image updates performed from within K8s. If we upgrade via helm, then I don't even know what it does to the configmap and without a way to test, we should simply skip this.